lib/events.js

```diff
@@ -353,8 +374,21 @@ class AdobeIOEvents {
             body: JSON.stringify(body),
             retryOptions: retryOptions
         });
-
-        return handleFetchErrors(response).json();
+        try {
+            response = handleFetchErrors(response).json();
+            return response;
+        }
+        catch (e) {
+            if (e.statusCode === 409) {
+                console.error("Journal already exists, returning existing");
+                const headers = response.headers.raw();
+                const conflictingRegistrationId = headers['x-conflicting-id'][0];
+                if (!headers || !conflictingRegistrationId) {
+                    throw e;
```

Comment:
There should be a logging statement here, it would help with debugging any issues.